README.md

```diff
@@ -98,7 +98,13 @@ All branches and pull requests and built and tested by Travis.
 
 For branches, the built assets are pushed to an analagous branch in [certbot/website-builds](https://github.com/certbot/website-builds). Built assets from PRs are not saved because Travis doesn't provide a mechanism to securely push to a Github repo after PRs across forks.
 
-To view the build of any branch, checkout that branch from certbot/website-builds and run some server to serve the files. For example,
+To view the build of any branch, checkout that branch from certbot/website-builds and run nginx to serve the files using the nginx configuration file from this certbot/website repository.
+
+A command to do this executed from the root of your local certbot/website-builds repository is
```

Comment:
```suggestion
For example, commands to do this might looks like:
A command to do this executed from the root of your local certbot/website-builds repository is
```